src/components/Dropdown/Dropdown.Props.ts

```diff
@@ -58,4 +58,9 @@ export interface IDropdownOption {
    * Whether this option is currently selected.
    */
   isSelected?: boolean;
+
+  /**
+   * Optional custom renderer for the dropdown item
+   */
+  onRender?: (item: IDropdownOption) => JSX.Element;

```

Comment:
Good point, I was going off of the ContextMenu, but that is a bit of a different application